fybrikapplication.yaml

```diff
@@ -7,7 +7,7 @@ spec:
    workloadSelector:
      matchLabels: {}
   appInfo:
-    intent: fraud-detection
+    intent: Fraud Detection
```

Comment:
you should quotes strings with spaces.

Reply:
OK btw I took it from https://fybrik.io/v0.5/samples/notebook/ - please see `Create a FybrikApplication resource for the notebook` section

Reply:
it has `intent: fraud-detection` as it was here, without spaces

Reply:
@roytman sorry I was referring to the dev version https://fybrik.io/dev/samples/notebook/

Reply:
I'd quote both cases, including the one in the docs